### DISCUSSION TITLE
How does woker notify ace that page initialization is complete

### DISCUSSION THREAD

**Discussion by ZhangGuangHuiCoder** [mkslanc/ace-linters#91] [Q&A] [answered]
## worker.ts
manager.setGlobalOptions(
    "typescript",
    {
        extraLibs: {
            cus: { content: cus.d.ts, version: 1 },
        },
    },
    true
);
the cus.d.ts from fetch('http://xx.code.d.ts') , I want fetch end Refresh page tips

**Comment by mkslanc** [comment, marked as answer]
If I understand you correctly, you want to add resolved ts declaration to `extraLibs` and trigger validation (or other lsp action). Right now, it should be done automatically - `setGlobalOptions` would call validation after worker serve this command and answer. [Request and setGlobalOptions example](https://github.com/mkslanc/ace-playground/blob/master/src/linters/linters.ts#L11)
